python/paddle/incubate/nn/layer/fused_transformer.py

```diff
@@ -824,7 +880,30 @@ def __init__(
             linear2_bias_attr=bias_attrs[1],
         )
 
-    def forward(self, src, src_mask=None, cache=None):
+    @overload
+    def forward(
+        self,
+        src: Tensor,
+        src_mask: Tensor | None = ...,
+        cache: Tensor | None = ...,
```

Comment:
```suggestion
        cache: None = ...,
```

这里的 `MultiHeadAttention.Cache` 是个 `namedtuple` ，其内部元素是 `Tensor` ～ 应该不是一个单独的 `Tensor` 作为输入 ～ 

p.s. 从代码上分析，应该是 `Tensor` 而不是 `MultiHeadAttention.Cache` ，先按照文档说明来吧  ～

Reply:
收到